snuba/stateful_consumer/states/bootstrap.py

```diff
@@ -14,12 +22,60 @@ class BootstrapState(State[ConsumerStateCompletionEvent, ConsumerStateData]):
     control topic.
     """
 
+    def __init__(self,
+        topic: str,
+        bootstrap_servers: Sequence[str],
+        group_id: str,
+    ):
+        super(BootstrapState, self).__init__()
+
+        def on_partitions_assigned(
+            consumer: Consumer,
+            partitions: Sequence[TopicPartition],
+        ):
+            pass
+
+        def on_partitions_revoked(
+            consumer: Consumer,
+            partitions: Sequence[TopicPartition],
+        ):
+            pass
```

Comment:
Are these `on_partitions_assigned` and `on_partitions_revoked` placeholders to be filled in later? If not, can we make the corresponding arguments to `StrictConsumer` `Optional`s?

Reply:
This makes sense. I was planning to use them, but later on I ended up leaving empty.